src/core/statistics/statistics.ts

```diff
@@ -21,7 +22,12 @@ export const statistics: Statistics = {
   reservation: { id: 50, name: "Reserver" },
   onlineReservation: { id: 51, name: "Se online" },
   autosuggestClick: { id: 54, name: "Autosuggest - klik" },
-  campaignShown: { id: 62, name: "KampagnePlus Titel" }
+  campaignShown: { id: 62, name: "KampagnePlus Titel" },
+
+  // Loaner status, user profile
+  renewSelectedMaterials: { id: 55, name: "Forny valgte materialer" },
+  renewAllMaterials: { id: 56, name: "Forny alle materialer" },
+  addToFavorites: { id: 55, name: "Tilføj til liste" }
```

Comment:
Shouldn't the id of "Tilføj til liste" be 61 instead of 55?

Reply:
you are absolutely right! 💯